src/book/03-guides/04-general/04-code-conventions.mdx

```diff
@@ -1,12 +1,48 @@
 ---
 section: Guides
 chapter: General
-title: Code Documentation
-description: How to use documentation tags
-slug: /guides/general/documentation
+title: Code Conventions
+description: Conventions used in NUbots code.
+slug: /guides/general/code-conventions
 ---
 
-## Conventions
+## Naming Conventions
+
+### Case
+
+NUbots code case conventions are in the below table. It does not include JavaScript, which is in the following table.
+
+| Scenario      | Case Type            | Example                |
+| ------------- | -------------------- | ---------------------- |
+| Class name    | PascalCase           | `MyClass`              |
+| Function name | snake_case           | `my_function()`        |
+| Variable name | snake_case           | `my_variable`          |
+| Constants     | SCREAMING_SNAKE_CASE | `MY_CONSTANT_VARIABLE` |
+| `yaml` fields | snake_case           | `my_field: 0.0`        |
+
+JavaScript conventions are in the below table.
```

Comment:
JS includes TS in this context, I would assume, but specifying it as JS could introduce ambiguity perhaps? Am I reading too far into that?

Reply:
TS _is_ JS. But I can specify if it's useful.

Reply:
like, yes, you're right 🤷